src/components/VExpansionPanel/VExpansionPanel.js

```diff
@@ -33,10 +32,38 @@ export default {
     }
   },
 
+  computed: {
+    classes () {
+      return {
+        'v-expansion-panel--focusable': this.focusable,
+        'v-expansion-panel--popout': this.popout,
+        'v-expansion-panel--inset': this.inset,
+        ...this.themeClasses
+      }
+    }
+  },
+
   watch: {
-    expand (v) {
-      this.open = Array(this.items.length).fill(false)
-      this.$emit('input', v ? this.open : null)
+    expand (val) {
+      let open
+      if (val) {
```

Comment:
could this just compare `!val` and perform the current **else** as its only conditional